<h3>Public RNA-seq Allele-Specific Expression (ASE) browser</h3>
<p>We have performed Allele Specific Expression (ASE) mapping on 1,262 human public RNA-seq samples (<a href="http://dx.doi.org/10.1101/007633" target="_blank">manuscript preprint</a>). We have identified 71,214 genetic variants showing significant (FDR ? 0.05) ASE. These variants can be queried using this website.</p>
<p>We have also made the other results and datasets from our manuscript available.</p>
<h4>ASE browser</h4>
<p>Click&nbsp;<a href="http://molgenis27.target.rug.nl/dataexplorer?dataset=ASE">here</a>&nbsp;to browse all significant ASE effects or use the search box below. Note: all positions are build 37 coordinates</p>
<p>Search options:</p>
<ul>
<li>Ensembl gene identifier (ENSG00000009724)</li>
<li>Gene symbol (MASP2)</li>
<li>SNP RS number (rs72550870)</li>
<li>Chrom:Pos (1:11106666)</li>
<li>Chrom:Start-Stop (1:11100000-11200000)</li>
</ul>
<div id="direct-repository-query-container" class="input-append col-md-4">
	<div class="input-group">
		<input id="direct-repository-query" class="span4" type="text" placeholder="Search for ASEs" />
		<span class="input-group-btn">
			<button class="btn btn-default" type="button" id="direct-repository-query-button"><span class="glyphicon glyphicon-search"></span></button>
		</span>
	</div>
</div>
<p><br /><br /></p>
<h4>eQTL results</h4>
<ul>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/eQtlGenesBladderFdr0.05.txt">Bladder eQTLs (FDR ? 0.05)</a></li>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/eQtlGenesBrainFdr0.05.txt">Brain eQTLs (FDR ? 0.05)</a></li>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/eQtlGenesBreastFdr0.05.txt">Breast eQTLs (FDR ? 0.05)</a></li>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/eQtlGenesLiverFdr0.05.txt">Liver eQTLs (FDR ? 0.05)</a></li>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/eQtlGenesAllSamplesFdr0.05.txt">All samples eQTLs (FDR ? 0.05)</a></li>
</ul>
<h4>Data for the 4,978 samples used for expression clustering</h4>
<ul>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/expression_table.genelevel.v71.htseq.txt.gz">Raw expression matrix</a></li>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/expression_table.genelevel.v71.htseq.QuantileNormalized.Log2Transformed.ProbesCentered.SamplesZTransformed.CovariatesRemoved.txt.gz">Normalized covariate corrected expression matrix</a></li>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/expression_table.genelevel.v71.htseq.QuantileNormalized.Log2Transformed.ProbesCentered.SamplesZTransformed.CovariatesRemoved.PCAOverSamplesEigenvectors.txt.gz">Expression PCA - Eigenvectors</a></li>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/expression_table.genelevel.v71.htseq.QuantileNormalized.Log2Transformed.ProbesCentered.SamplesZTransformed.CovariatesRemoved.PCAOverSamplesPrincipalComponents.txt.gz">Expression PCA - Principal components</a></li>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/expression_table.genelevel.v71.htseq.QuantileNormalized.Log2Transformed.ProbesCentered.SamplesZTransformed.CovariatesRemoved.PCAOverSamplesEigenvalues.txt.gz">Expression PCA - Eigenvalues</a></li>
</ul>
<h4>Data for the 1,262 unique samples used for ASE and eQTL analyses</h4>
<ul>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/expression_table.genelevel.v71.htseq.batch9.txt.gz">Raw expression matrix</a></li>
<li><a href="https://molgenis26.target.rug.nl/downloads/publicRnaSeq/expression_table.genelevel.v71.htseq.batch9.TMM.ProbesWithZeroVarianceRemoved.Log2Transformed.ProbesCentered.SamplesZTransformed.CovariatesRemoved.100PCAsOverSamplesRemoved.txt.gz">Normalized expression matrix</a></li>
</ul>
<p>The genotype data is available upon request, please e-mail&nbsp;&nbsp;patrick&nbsp;&nbsp;deelen&nbsp;&nbsp;at&nbsp;&nbsp;gmail&nbsp;dot&nbsp;com or dasha&nbsp;&nbsp;zhernakova&nbsp;&nbsp;at&nbsp;&nbsp;gmail&nbsp;dot&nbsp;com</p>